### PR TITLE
Delete resources created during this task

### PR DESCRIPTION
**Pull request by oussemos**

### CODE REVIEW

File: content/en/docs/tasks/configure-pod-container/configure-projected-volume-storage.md [hunk 1]
```diff
@@ -67,6 +67,16 @@ the Pod:
...
     ```
+
+## Clean up
+
+Delete the Pod and the Secrets:
+
+```shell
+kubectl delete pod test-projected-volume
+kubectl delete secret user pass
+```
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/configure-projected-volume-storage.md]: Is it worth nudging the reader to remove `username.txt` and `password.txt` as well?

**Comment by tengqm** [comment]
I believe cleaning resources is enough.
/lgtm

File: content/en/docs/tasks/configure-pod-container/configure-projected-volume-storage.md (hunk 1 above)
Comment (oussemos) [reply content/en/docs/tasks/configure-pod-container/configure-projected-volume-storage.md]: As @tengqm said, I also believe cleaning k8s resources is enough. People following this tutorial should have already the minimum of Linux knowledge to `rm` the created files.

**Pull request merged**